apps/ewallet/test/ewallet/gates/transaction_consumption_gate_test.exs

```diff
@@ -191,6 +191,49 @@ defmodule EWallet.TransactionConsumptionGateTest do
 
       assert res == {:error, :account_balance_mismatch}
     end
+
+    test "works for account even if max_consumptions_per_user is set", meta do
```

Comment:
What's the rationale for accounts bypassing this rule? Just for my own knowledge.

Reply:
Good point, removed it in the refactor.